### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by yfarjoun**
Add fgprimer to landing page

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -25,6 +27,7 @@ Here are some _libraries_ that we use regularly in our work:
 * Python
   * [fgpyo](https://github.com/fulcrumgenomics/fgpyo): Quality of life improvements for Bioinformatics in Python.
   * [pybedlite](https://github.com/fulcrumgenomics/pybedlite): Lightweight Python interfaces for reading, writing, and querying Genomic Regions (BED).
+  * [fgprimer](https://github.com/fulcrumgenomics/fgprimer): APIs and wrappers for performing PCR primer design related tasks. Uses primer3, ntThal and bwa.
```
Comment (nh13) [inline profile/README.md:30]: See line 37

**Review by nh13 — CHANGES_REQUESTED** [review]
Don't we already have this?

File: profile/README.md (hunk 1 above)
Comment (nh13) [reply profile/README.md:30]: https://github.com/fulcrumgenomics/.github/blob/688e1aeea60cc90c98107f26587a783fa1e89602/profile/README.md?plain=1#L34

Comment (yfarjoun) [reply profile/README.md:30]: sorry.

**Pull request closed without merging**